packages/flutter/test/widgets/form_test.dart

```diff
@@ -204,8 +204,8 @@ void main() {
     expect(editableText.widget.controller.text, equals('world'));
   });
 
-  testWidgets('Provide initial value to input when controller is specified', (WidgetTester tester) async {
```

Comment:
Don't you still want to *also* test that the initialValue gets set on the implicitly created controller?

Reply:
Good point, yes.